package.json

```
@@ -127,7 +127,7 @@
     "lodash": "^4.17.19",
     "loglevel": "^1.4.1",
     "luxon": "^1.24.1",
-    "metamask-logo": "^2.1.4",
+    "metamask-logo": "^2.4.0",
```

It looks like `v2.4.1` is what is used in the lockfile. What is the difference between v2.4.0 and v2.4.1 :thinking: The git history is a bit confusing here because of the `master`/`gh-pages` split in that repo - perhaps you know @rekmarks ?

```suggestion
    "metamask-logo": "^2.4.1",
```

---

@danfinlay published `2.4.1` minutes after `2.4.0`, and the `2.4.0` PR actually pushed to `2.4.1`. I'm not sure what the difference is, if any, but I think we should definitely use the latter, as is now the case.